tools/custom-checks/check-dependency-version-consistency.ts

```diff
@@ -0,0 +1,15 @@
+import { CDVC } from "check-dependency-version-consistency";
+
+export async function checkDependencyVersionConsistency(): Promise<
+  [boolean, string[]]
+> {
+  const errors: string[] = [];
+  const cdvc = new CDVC(process.cwd(), {
+    fix: false,
+  });
+  errors.push(cdvc.toMismatchSummary());
```

Comment:
@jagpreetsinghsasan If there is a way to extract the list of issues one by one that would work best (but I don't know if the library itself supports this or not).
If we can only get the summary then it's fine by me like this, BUT if we can get an array or other collection of individual issues instead of just a summary, then I'd loop through those and add a separate error for each. 
This will (hopefully) aid us in interpreting the results more easily later on and it could also allow us the chance to add extra details for each error (I don't know what the output of the summary looks like, I'm just speculating that it may not contain all the relevant details needed to fix the problems such as the exact location of the package.json file where the problem dependency is declared, or the version it is declared with ,etc.)

Reply:
Yes, you are right @petermetz. I straightaway pushed the summary because of the nature of the result it provides.
The library outputs this as a single, highly formatted string with tables and color encoding.
![image](https://github.com/user-attachments/assets/eaedcf45-3fa5-460c-aa5e-9c1c72211e8d)

Reply:
@jagpreetsinghsasan OK, thank you for confirming, in that case, I'm OK with just leaving it as is.